### PR TITLE
adds filter & pagination function

### PR DESCRIPTION
**Pull request by vivex**
https://github.com/shippablepmtool/pmwww/issues/5

### CODE REVIEW

File: src/issue-tracker/issue-tracker.html [hunk 1]
```diff
@@ -10,13 +10,26 @@
                 <div class="input-group">
                     <input type="text" class="form-control"
                            placeholder="Organisation Name" ng-model="org"
+                           ng-disabled="showSpinner"
                            required>
       <span class="input-group-btn">
```
Comment (harryi3t) [inline src/issue-tracker/issue-tracker.html]: indentation of `<span>`: move it under the `<div>` on line 10

File: src/issue-tracker/issue-tracker.html [hunk 2]
```diff
@@ -25,94 +38,194 @@ <h3 class="panel-title">Open Issues</h3>
         </div>
         <div class="panel-body">
             <div class="row">
-                <div class="col-lg-6 col-md-6 col-sm-12">
-                    <h2>Total Issue: 20</h2>
-
+                <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12">
+                    <h2>Total Issue: {{filteredIssueList.length}}</h2>
```
Comment (harryi3t) [inline src/issue-tracker/issue-tracker.html]: Should be `Total Issues:`

File: src/issue-tracker/issue-tracker.html [hunk 3]
```diff
@@ -25,94 +38,194 @@ <h3 class="panel-title">Open Issues</h3>
...
+                                <th><a href="#"
+                                       ng-click="sortType = 'assignee'; sortReverse = !sortReverse">
+                                    Assignee
+                                    <span ng-show="sortType == 'assignee' && !sortReverse"
+                                          class="glyphicon glyphicon-triangle-bottom sort-icon"></span>
+                                    <span ng-show="sortType == 'assignee' && sortReverse"
+                                          class="glyphicon glyphicon-triangle-top sort-icon"></span>
+                                </a></th>
+
+                                <th class="hidden-sm hidden-xs"><a href="#"
```
Comment (harryi3t) [inline src/issue-tracker/issue-tracker.html]: see if you can move the anchor tag on new line. Also on line 154 move the `th` on new line

File: src/issue-tracker/issue-tracker.html (hunk 1 above)
Comment (vivex) [inline src/issue-tracker/issue-tracker.html]: Done.

File: src/issue-tracker/issue-tracker.html (hunk 2 above)
Comment (vivex) [inline src/issue-tracker/issue-tracker.html]: Done.

File: src/issue-tracker/issue-tracker.html (hunk 3 above)
Comment (vivex) [inline src/issue-tracker/issue-tracker.html]: Done

**Comment by harryi3t** [comment]
@vivex hey could you post the screenshots also (for large, medium & extra-small devices)

**Comment by vivex** [comment]
Large Screen View:
large_screen_view_pm

Small Screen View:
mobile_view_pm
(Table is horizontally scroll-able )

**Pull request merged**